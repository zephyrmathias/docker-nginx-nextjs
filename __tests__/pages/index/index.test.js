import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'
import Homepage from '../../../pages/index'

describe('snapshot', () => {
  it('should match with snapshot', () => {
    const component = shallow(<Homepage />)
    expect(component).toMatchSnapshot()
  })
})
