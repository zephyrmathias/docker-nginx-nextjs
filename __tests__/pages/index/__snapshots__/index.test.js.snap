// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot should match with snapshot 1`] = `
<index__Header>
  Homepage
</index__Header>
`;
